AIPscan/Aggregator/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Question noted in the PR comment: `Ideally we would want to define fixtures such as app_instance once rather than separately in each Blueprint,` yeah, it does seem like an unnecessary redundancy but I'm comfortable with it for now - especially if it means not co-locating the tests. Also, suggested solution noted. IDK myself yet. Let's have a think. This work is definitely two steps forward so that's :ok_hand: 

One thing I would love to see modified slightly is the naming of this `conftest.py` and the other one. Primarily as it isn't a test, and secondly because it loses its snake casing so it's a bit of a portmanteau of sorts. Off the top off my head - `app_fixtures.py`?

Reply:
> One thing I would love to see modified slightly is the naming of this conftest.py and the other one.

I fully agree and would love to give it a better name, but unfortunately the name is [built-in to pytest](https://docs.pytest.org/en/stable/fixture.html?highlight=conftest#conftest-py-sharing-fixture-functions) and the fixture won't be auto-discovered if we change the module name.

>  yeah, it does seem like an unnecessary redundancy but I'm comfortable with it for now - especially if it means not co-locating the tests. Also, suggested solution noted. IDK myself yet. Let's have a think.

After going through the pytest documentation again, I think it may actually be possible to remove the duplication by moving the `conftest.py` file into the main `AIPscan` package. Will test that out and see if it gives us another potential solution that doesn't require moving the tests around...

Reply:
> I fully agree and would love to give it a better name, but unfortunately the name is built-in to pytest and the fixture won't be auto-discovered if we change the module name.

TIL!!

Reply:
It was new to me too! The top answer on this SO post gives a good overview of all the things that can be done with `conftest.py`: https://stackoverflow.com/questions/34466027/in-pytest-what-is-the-use-of-conftest-py-files